redux/actions/currenciesActions.ts

```diff
@@ -1,13 +1,14 @@
 import AsyncStorage from '@react-native-async-storage/async-storage';
 import { Dispatch } from 'redux';
-import { IGetCurrenciesAction } from '../../interfaces';
-import { GET_USER_CURRENCIES_FROM_ASYNC } from '../actionsTypes';
+import { ICurrency, IGetCurrenciesAction } from '../../interfaces';
+import { ADD_NEW_CRYPTOCURRENCY, GET_USER_CURRENCIES_FROM_ASYNC } from '../actionsTypes';
+import { IState } from '../reducers/mainReducer';
 
 export const getUserCurrenciesFromAsync = () => {
   return async (dispatch: Dispatch<IGetCurrenciesAction>) => {
     try {
       const userCurrenciesList = await AsyncStorage.getItem('@userCurrenciesList')
-      if (userCurrenciesList != null ) {
+      if (userCurrenciesList) {
```

Comment:
[minor] There should be a validation for the length here, since you can save an empty array if all items are removed from it